sdk/src/claim.rs

```diff
@@ -1152,32 +1149,33 @@ impl Claim {
         match verified {
             Ok(vi) => {
                 if !vi.validated {
-                    let log_item = log_item!(
+                    log_item!(
                         claim.signature_uri(),
                         "claim signature is not valid",
                         "verify_internal"
                     )
-                    .error(Error::CoseSignature)
-                    .validation_status(validation_status::CLAIM_SIGNATURE_MISMATCH);
-                    validation_log.log(log_item, Some(Error::CoseSignature))?;
+                    .validation_status(validation_status::CLAIM_SIGNATURE_MISMATCH)
+                    .failure(validation_log, Error::CoseSignature)?;
                 } else {
-                    let log_item = log_item!(
+                    log_item!(
                         claim.signature_uri(),
                         "claim signature valid",
                         "verify_internal"
                     )
-                    .validation_status(validation_status::CLAIM_SIGNATURE_VALIDATED);
-                    validation_log.log_silent(log_item);
+                    .validation_status(validation_status::CLAIM_SIGNATURE_VALIDATED)
+                    .success(validation_log);
                 }
             }
             Err(parse_err) => {
-                let log_item = log_item!(
+                log_item!(
                     claim.signature_uri(),
                     "claim signature is not valid",
                     "verify_internal"
                 )
-                .error(parse_err);
-                validation_log.log(log_item, None)?;
+                // .validation_status(???)
+                // PR REVIEW: Should there be a validation status?
```

Comment:
@mauricefisher64 there wasn't a validation status here before. Should there have been?